packages/pg/package.json

```diff
@@ -11,43 +11,55 @@
     "postgresql",
     "rdbms"
   ],
-  "homepage": "https://github.com/brianc/node-postgres",
+  "homepage": "git://github.com/daniel-liu-bitio/node-postgres.git",
   "repository": {
     "type": "git",
-    "url": "git://github.com/brianc/node-postgres.git",
+    "url": "git://github.com/daniel-liu-bitio/node-postgres.git",
     "directory": "packages/pg"
   },
-  "author": "Brian Carlson <brian.m.carlson@gmail.com>",
-  "main": "./lib",
-  "dependencies": {
-    "buffer-writer": "2.0.0",
-    "packet-reader": "1.0.0",
-    "pg-connection-string": "^2.5.0",
-    "pg-pool": "^3.5.1",
-    "pg-protocol": "^1.5.0",
-    "pg-types": "^2.1.0",
-    "pgpass": "1.x"
```

Comment:
Context: I don't understand npm/bundling. Why are there only devDependencies, now?

Reply:
I think for our purpose, dependencies and devDependencies are the same since they get bundled in the same way through webpack. I changed some packages back to regular dependencies, though, so if you clone the repo then run "npm install", all the regular dependencies listed will be installed.